docOps/docs/libraries/rust/getting_started.md

```diff
@@ -0,0 +1,87 @@
+# Getting Started with Rust
+
+## Prerequisites
+
+ _Rust_  and  _Cargo_  are required to use Wallet.rs. You can find installation instructions in the [Rust documentation](https://doc.rust-lang.org/cargo/getting-started/installation.html).
+
+We recommend you update  _Rust_  to the latest stable version [rustup update stable](https://github.com/rust-lang/rustup.rs#keeping-rust-up-to-date). The nightly version should be fine, but there is a chance some changes are not compatible.
+
+ _no_std_  is not currently supported, but we are working on it, and we will provide it as a feature once the new implementation is ready.
+
+### Dependencies
+
+ _cmake_  and  _openssl_  are required. In order to run the build process successfully using Cargo you may need install additional build tools on your system. 
+
+### Windows
+
+ _cmake_  can be downloaded from the [official cmake website](https://cmake.org/download/).
+ _openssl_  can be installed with [vcpkg](https://github.com/microsoft/vcpkg) or [chocolatey](https://chocolatey.org/).
+
+- Installing  _openssl_  with  _vcpkg_ :
+
+    ```
+    $ ./vcpkg.exe install openssl:x64-windows
+    $ ./vcpkg.exe integrate install
+    # you may want to add this to the system environment variables since you'll need it to compile the crate
+    $ set VCPKGRS_DYNAMIC=1
+    ```
+
+- Installing  _openssl_  with  _chocolatey_ :
+
+    ```
+    $ choco install openssl
+    # you may need to set the OPENSSL_ROOT_DIR environment variable
+    $ set OPENSSL_ROOT_DIR="C:\Program Files\OpenSSL-Win64"
+    ```
+
+### macOS
+
+ _cmake_  and  _openssl_  can be installed with  _Homebrew_  by running the following commands:
+
+```
+$ brew install cmake
+$ brew install openssl@1.1
+# you may want to add this to your .zshrc or .bashrc since you'll need it to compile the crate
+$ OPENSSL_ROOT_DIR=$(brew --prefix openssl@1.1)
+```
+
+### Linux
+
+You can install  _cmake_  and  _openssl_  with your distro's package manager or download from their websites. On Debian and Ubuntu you will also need the `build-essential` meta-package.
+
+## Usage
+
+To use the library, add this to your  _Cargo.toml_ :
+
+```
+[dependencies]
+iota-wallet = { git = "https://github.com/iotaledger/wallet.rs" }
```

Comment:
this is dangerous, because then they are pulling from whatever default branch the IF chooses. Please specify a branch here, a tag, or a git revision.

Reply:
I've set the main branch, please let me know if I should update it to another.

Reply:
Where? In the repo settings? That is opaque for a consumer using git via automation or developer tooling. (Or is my GH interface not showing latest changes?)

```
iota-wallet = { git = "https://github.com/iotaledger/wallet.rs", branch = "main" }
```